scripts/scil_surface_create.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to create a surface with marching cube from a mask or a label image.
+The surface will be readable with software like MI-Brain.
+
+Example : use wmparc.a2009s.nii.gz with some aseg.stats indices
+
+scil_surface_create.py out_surface.vtk \\
+    --in_labels s1a1/mask/S1-A1_wmparc.a2009s.nii.gz\\
+    --list_indices 16:32 --opening 2 --smooth 2 -v
+"""
+
+import argparse
+import logging
+import os
+
+import nibabel as nib
+import numpy as np
+import trimeshpy
+import trimeshpy.vtk_util as vtk_u
+from scipy.ndimage import (binary_closing,
+                           binary_dilation,
+                           binary_erosion,
+                           binary_opening,
+                           binary_fill_holes)
+import mcubes
+
+from scilpy.image.labels import (get_data_as_labels,
+                                 merge_labels_into_mask)
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             ranged_type)
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    g1 = p.add_argument_group("Input (Labels or Mask)")
+    mxg = g1.add_mutually_exclusive_group(required=True)
+    mxg.add_argument('--in_labels',
+                     help='Path of the atlas (nii or nii.gz).\n'
+                          'You can provide a list of indices with '
+                          '--list_indices or create a surface per '
+                          'index with --each_index.\n'
+                          'If no indices are provided, '
+                          'it will merge all indices and converted '
+                          'to a binary mask.')
+    mxg.add_argument('--in_mask',
+                     help='Path of the mask (nii or nii.gz).')
+    mxg.add_argument('--in_volume',
+                     help='Path of the volume (nii or nii.gz).')
+
+    p.add_argument('out_surface',
+                   help='Output surface (.vtk)')
+
+    g2 = p.add_argument_group("Options for labels input")
+    mxg2 = g2.add_mutually_exclusive_group()
+    mxg2.add_argument('--list_indices', nargs='+',
+                      help='List of labels indices to use for the surface.')
+    mxg2.add_argument('--each_index', action='store_true',
+                      help='Create a surface per index. It will use the '
+                           'out_surface basename to create the output files.')
+
+    g3 = p.add_argument_group('Options for volume input')
+    g3.add_argument('--value', default=0.5,
```

Comment:
Since this value is force to 0.5 later on, I suggest to remove the args.value = 0.5 at line  # 129, and # 148
otherwise, remove this option and use a "global constant"